### PR TITLE
[EMCAL-554] Development of a class containing the trigger DCS information in the CCDB

### PR DESCRIPTION
**Pull request by hahassan7**
Development of a container class for the trigger information received from the DCS (TRU and STU) along with the unit tests, and the read write tests. Several containers were added: 

1. TriggerSTUDCS: container class for the STU information with unit tests.
2. TriggerTRUDCS: container class for the TRU information with unit tests.
3. TriggerSTUErrorCount: container class for the STU errors with unit tests.
4. TriggerDCS: container class for the DCS information with the unit tests and the read write tests.

### CODE REVIEW

File: Detectors/EMCAL/calib/macros/TriggerDCS_CCDBApiTest.C [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  dcs->setSTUEMCal(stuEMCal);
+  dcs->setSTUDCal(stuDCal);
+  dcs->setTRU(tru);
+
+  // Set time limits: These are from the start of the run validity range (252235) to the end of the run validity range (267166) LHC16
+  auto rangestart = create_timestamp(2016, 4, 23, 0, 58, 40),
+       rangeend = create_timestamp(2016, 12, 5, 6, 3, 19);
+  std::cout << "Using time stamps " << rangestart << " and " << rangeend << std::endl;
+  std::map<std::string, std::string> metadata;
+  ccdbhandler.storeAsTFile(new o2::TObjectWrapper<o2::emcal::TriggerDCS>(dcs), "EMC/TriggerDCS", metadata, rangestart, rangeend);
```
Comment (sawenzel) [inline Detectors/EMCAL/calib/macros/TriggerDCS_CCDBApiTest.C]: TObjectWrapper is no longer needed! Can you please now use the `ccdbhandler.storeAsTFileAny` interface which allows you to store any type.

**Comment by hahassan7** [comment]
Hi @sawenzel , I removed everything related to TObjectWrapper.

**Comment by sawenzel** [comment]
There are still some `#includes` for TObjectWrapper. Once removed, this can be merged.

**Comment by hahassan7** [comment]
Hi @sawenzel, sorry I forgot the includes. Now they are all removed.

**Pull request merged**